### ISSUE TITLE
Loss metric and pretraining

### ISSUE THREAD

**Issue by tarunn2799** [google-research/simclr#64]
I'm trying to run the code on colab for pre-training cifar10  using an NVIDIA P100. 
I have two quick doubts:

1. Why is the loss metric global_step/sec and not contrastive loss? This issue was there in BERT as well, and they'd mentioned it was because of the tpu_estimator.py (even though I'm running single gpu).  How do I change this to display different loss metrics?

2. I ran it for 1 epoch with a batch size of 128, and it still ran for 30+ minutes. Any idea why it took that long, because it sounds unusual to run that long for a small dataset like cifar10.

**Comment by chentingpc** [comment]
1. you could try to set `train_summary_steps` to enable tensorboard logging.
2. are you sure it is using GPU correctly?

**Comment by tarunn2799** [comment]
1. alright, i'll try that and get back to you,
2. I used the command present in the repository to train on single gpu, in colab (with use_tpu=False). 

It is using my gpu, I was able to check on the side. A P100 is good enough, right?
